entity-service/src/main/resources/application.properties

```diff
@@ -11,8 +11,8 @@ spring.security.oauth2.resourceserver.jwt.jwk-set-uri = https://data-hub.eglobal
 
 server.port = 8082
 
-management.endpoints.web.exposure.include = health,info,metrics,bindings
-management.health.db.enabled = false
```

Comment:
Does the added properties disable the db health check ?
Since we had a deployment problem related to this

Reply:
We had a problem with the migration tool you removed since ...